src/Field/Container.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponentElement } from 'react'
+import { FieldProps } from './Field'
+
+type FieldElement = FunctionComponentElement<any>
+
+interface ContainerProps extends FieldProps {
+  children: FieldElement | FieldElement[]
+}
+
+const Container: React.FC<ContainerProps> = ({ children, ...otherProps }) => {
+  return <>
```

Comment:
Why not render this in a div? Now you have to use Container as a child somewhere.

Reply:
Or perhaps make it a form?

Reply:
Let's make it a `div`. I'll pick it up (or you do it) in a later PR okay?